src/spryker/harness.yml

```diff
@@ -89,7 +89,7 @@ attributes:
       php-fpm: "= 'my127/spryker:' ~ @('php.version') ~ '-fpm-' ~ (@('php.version') >= 7.4 ? 'buster' : 'stretch')"
   elasticsearch:
     image: elasticsearch
-    tag: '6.8.12'
+    tag: '7.16.2'
```

Comment:
not all spryker projects will work with elasticsearch 7, though there's 6.8.22, which I'm changing to in another PR.

Should we do spryker version check here?